From 518ba780b1dc96b53fe3507f0943777e850b7780 Mon Sep 17 00:00:00 2001
From: Dmitri Epshtein <dima@marvell.com>
Date: Mon, 18 Jul 2016 18:01:54 +0300
Subject: [PATCH 0611/2241] arm: defconfig: enable VLAN support for armv7

Add CONFIG_VLAN_8021Q=y to the default kernel configuration
- arch/arm/config/mvebu_v7_lsp_defconfig

Change-Id: I146c4856b2c11f3ca494ad215839aa548009f809
Signed-off-by: Dmitri Epshtein <dima@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31221
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Nir Erez <nerez@marvell.com>
Reviewed-by: Yan Markman <ymarkman@marvell.com>
Reviewed-by: Lior Amsalem <alior@marvell.com>
---
 arch/arm/configs/mvebu_v7_lsp_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/mvebu_v7_lsp_defconfig b/arch/arm/configs/mvebu_v7_lsp_defconfig
index 5101bee..702d127 100644
--- a/arch/arm/configs/mvebu_v7_lsp_defconfig
+++ b/arch/arm/configs/mvebu_v7_lsp_defconfig
@@ -41,6 +41,7 @@ CONFIG_IP_PNP=y
 CONFIG_IP_PNP_DHCP=y
 CONFIG_IP_PNP_BOOTP=y
 CONFIG_BRIDGE=y
+CONFIG_VLAN_8021Q=y
 CONFIG_BT=y
 CONFIG_BT_MRVL=y
 CONFIG_BT_MRVL_SDIO=y
-- 
2.7.4

